config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```
@@ -520,25 +456,24 @@ fields:
                 base_table: project
                 prop_table: projectprop
         settings:
-            termIdSpace: EFO
-            termAccession: '0001736'
+            termIdSpace: local
+            termAccession: 'project_germcollection'
         display:
           view:
             default:
               region: content
               label: above
-              weight: 22
+              weight: 20
           form:
             default:
               region: content
-              weight: 22
+              weight: 20
 
-    # @todo update to chado_pub_type_default with type specified Chado#140
-    -   name: experiment_citation
-        content_type: greenhouse_experiment
-        label: Data Citation
+    -   name: field_featureofinterest
+        content_type: field_experiment
+        label: Feature of Interest
```

While this is the name of the term it is not really intuitive as the name of the field presented to the users. I went with core measurements so that we are not 100% restricted to phenotypes and measurements instead of traits because we want them to be very specific like our examples are. The trait is just shattering but the measurements are the shattered seed weight or a scale.
```suggestion
        label: Core Measurements
```